docs/docs/producers.md

```diff
@@ -117,3 +117,17 @@ Set the output stream in `config.properties` by setting the `kinesis_stream` pro
 The producer uses the [KPL (Kinesis Producer Library](http://docs.aws.amazon.com/streams/latest/dev/developing-producers-with-kpl.html) and uses the KPL built in configurations.
 Copy `kinesis-producer-library.properties.example` to `kinesis-producer-library.properties` and configure the properties file to your needs.
 The most important option here is configuring the region.
+
+### SQS AWS credentials
+***
+You will need to obtain an IAM user that has the permission to access the SQS service. The sqs producer also uses [DefaultAWSCredentialsProviderChain](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html) to get aws credentials.
+
+See the [AWS docs](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html) on how to setup the IAM user with the Default Credential Provider Chain.
+
+In case you need to set up a different region also along with credentials then default one, see the [AWS docs](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html#setup-credentials-setting-region).
+
+### SQS Options
+***
+Set the output queue in the `config.properties` by setting the `sqs_queue_uri` property to full SQS queue uri from aws console.
+
+The producer uses the [AWS SQS SDK](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/sqs/AmazonSQSClient.html).
```

Comment:
Nitpicks on the doc: `sqs` -> `SQS`, `aws` -> `AWS` (avoid mixed casing for product names)

Reply:
resolved